File: xml-performance/test-drive-xml-main-runner/src/main/java/org/jesperancinha/test/drive/books/XmlTestRunner.java
/**
 * 
 */
package org.jesperancinha.test.drive.books;

/**
 * @author joao
 *
 */
public class XmlTestRunner {

	/**
	 * @param args
	 * @throws Exception
	 */
	public static void main(String[] args) throws Exception {
		final XmlTestRunnerWorker worker = new XmlTestRunnerWorker(args);
		if (!worker.isNoop()) {
			worker.runTests();
		}
	}
}

File: xml-performance/test-drive-xml-main-runner/src/main/kotlin/org/jesperancinha/test/drive/books/XmlTestRunner.kt
/**
 *
 */
package org.jesperancinha.test.drive.books

/**
 * @author joao
 */
fun main(args: Array<String>) {
    val worker = XmlTestRunnerWorker(args)
    if (!worker.isNoop) {
        worker.runTests()
    }
}